docs/docs/cmd/tenant/report/report-settings-set.mdx

```diff
@@ -0,0 +1,69 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# report settings set
+
+Update (adminReportSettings) tenant-level settings for Microsoft 365 reports
+
+## Usage
+
+```sh
+m365 tenant report settings set [options]
+```
+
+## Options
+
+```md definition-list
+`-h, --hideUserInformation  <hideUserInformation>`
+: Determines whether all reports conceal user information such as usernames, groups, and sites.  
+
+`--verbose`
+: Runs command with verbose logging
+
+`--debug`
+: Runs command with debug logging
+
+```
+
+<Global />
+
+## Remarks
+
+Please refer to the [Graph documentation page](https://learn.microsoft.com/en-in/graph/api/adminreportsettings-update?view=graph-rest-1.0&tabs=http).
```

Comment:
Let's remove this link. It's not necessary to refer API documentation that we use. We only add links that really add value for the end-user, like extra explanation about reports or something... It's not strictly necessary here I think though.